### DISCUSSION TITLE
Desktop Cruiser GL not start on Windows

### DISCUSSION THREAD

**Discussion by JonasR** [devemux86/cruiser#41] [Q&A] [answered]
Hi all, I didn't find any documentation on it, and it's been a while I used Java unfortunately.

I ran Cruiser 3.0.9 via the batch file on Windows and enabled OpenGL. Now the app doesn't even start anymore and just stops responding right away. This might be an unrelated issue with my system, but how can I remove the setting again?

I checked in `%AppData%` and didn't find anything. Not sure where else I should be looking.

Thanks a lot for your help!

**Comment by 0709wiwiwi** [comment]
Open by a previous version 3.07 and set Mapsforge.
 Continue than by next version. (Keep away from open GL until solved)

Op vr 17 feb. 2023 om 22:27 schreef JonasR ***@***.***>:

> Hi all, I didn't find any documentation on it, and it's been a while I
> used Java unfortunately.
>
> I ran Cruiser 3.0.9 via the batch file on Windows and enabled OpenGL. Now
> the app doesn't even start anymore and just stops responding right away.
> This might be an unrelated issue with my system, but how can I remove the
> setting again?
>
> I checked in %AppData% and didn't find anything. Not sure where else I
> should be looking.
>
> Thanks a lot for your help!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/devemux86/cruiser/discussions/41>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AM5INIPGBHBBAUX6MVUJKZTWX7UK7ANCNFSM6AAAAAAU74MADA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Willy
***@***.***

**Comment by JonasR** [comment]
Thank you for the super quick answer!
Github no longer hosts 3.0.7 it seems, where I can I still get the .zip? I tried with 3.0.8 and get the same problem.

**Reply by 0709wiwiwi** [reply]
Link to zip (dropbox) . (= Only temporary available !)
https://www.dropbox.com/s/n77bk82cxaeoyya/cruiser-3.0.7.zip?dl=0

**Reply by JonasR** [reply]
Thank you, I was able to reset with that version!

**Comment by devemux86** [comment]
There seems to be a problem with OpenGL on Windows, see https://github.com/devemux86/cruiser/discussions/40.

I will make another improved release soon. :slightly_smiling_face:

**Comment by devemux86** [comment]
You can make Cruiser start again with Mapsforge with the command:

```console
java -jar cruiser.jar mapsforge
```

**Reply by JonasR** [reply]
Thank you! My CPU's iGPU seems to be faulty and crashes in OpenGL apps all the time. I didn't even consider there might be an issue with Cruiser's OpenGL in parallel :)

**Reply by devemux86** [reply]
There has been a recent change in Cruiser to better run OpenGL on Mac (https://github.com/devemux86/cruiser/discussions/20).

Without success, so I will revert it in the next update (soon), as it also causes problems on Windows.

I also use an Intel GPU, no problems, but I work on Linux. :slightly_smiling_face:

**Comment by devemux86** [comment]
@JonasR 

Can you try with desktop [Cruiser 3.0.10 (Beta)](https://www.dropbox.com/sh/914t158im4b7lxr/AADOeSE1UjaW4FS2PLc7kwAma)?

**Reply by JonasR** [reply]
Yep, that one works with OpenGL 👍

**Comment by devemux86** [comment, marked as answer]
It should work properly with [Cruiser 3.0.10](https://github.com/devemux86/cruiser/releases).

**Reply by JonasR** [reply]
It does, thanks! :)
